docs/hacking/curriculum.rst

```diff
@@ -100,129 +49,6 @@ and enter this repository::
-building and testing curriculum contributions. As was briefly mentioned in the ``start`` section above, the ``selfmedicate``
-script takes care of mapping the files on your local filesystem into minikube and again into the Syringe pod to ensure
-it sees the lessons you're working on.
-
-This means you can work on lessons all on your local machine without having to bother editing environment variables or
-committing your content to a Git repository.
-
-Once you have a working antidote installation according to the previous section, you'll notice that the web portal shows the lessons
-as they existed when you initially started the platform. If you want to apply any changes you've made locally, you need to run the
-``reload`` subcommand of the ``selfmedicate`` script:
-
-.. code::
-
-    ./selfmedicate.sh reload
-
-This command will take care of restarting Syringe, so that it can reload the content you've changed on your filesystem.
-
-Pausing and Resuming Environment
---------------------------------
-
-As mentioned above, if you destroy the minikube environment, you'll need to perform the ``start`` command all over again.
-However, it would be nice to be able to stop the environment temporarily, and resume later without installing everything
-over again from scratch.
-
-Fortunately, the ``stop`` and ``resume`` subcommands do just this for us. To stop/pause the environment, run:
-
-.. code::
-
-    ./selfmedicate.sh stop
-
-To resume, run:
-
-.. code::
-
-    ./selfmedicate.sh resume
-
-The ``resume`` command is important to run, since this re-executes minikube with the optional arguments needed
-by Antidote, so make sure to use this, rather than trying to use ``minikube start`` directly.
-
-Troubleshooting Self-Medicate
------------------------------
-
-The vast majority of all setup activities are performed by the ``selfmedicate`` script. The idea is that this
-script shoulders the burden of downloading all the appropriate software and building is so that you can
-quickly get to focusing on lesson content.
-
-However, issues can still happen. This section is meant to direct you towards the right next steps should
-something go wrong and you need to intervene directly.
-
-.. warning::
-
-    The ``selfmedicate`` script is designed to make it easy to configure a local minikube environment
-    with everything related to Antidote installed on top. However, you'll always be well-served by
-    becoming familiar with ``minikube`` or even Kubernetes itself so that you are more able to troubleshoot
-    the environment when things go wrong. Keep a bookmark to the
-    `minikube docs <https://kubernetes.io/docs/setup/minikube/>`_ handy, just in case.
-
-.. note::
-
-    If your issue isn't covered below, please `open an issue on the
-    selfmedicate repository <https://github.com/nre-learning/antidote-selfmedicate/issues/new>`_.
-
 Cannot connect to the Web Front-End
```

Comment:
This is fine for now but I'll be following up with https://github.com/nre-learning/antidote-selfmedicate/issues/37 soon which will provide over-arching troubleshooting steps. I'll open a corresponding docs PR to cover that.

Reply:
I am leaving this in as you noted it is fine. Once the PR is in for the troubleshooting steps, this page can be updated.

Reply:
:+1: